### DISCUSSION TITLE
Issue in using pbeis

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#4244] [Q&A] [answered]
I am trying to implement EIS using the pbeis library, but I am getting an error that it requires an older version of pybamm (22.10), and even after installing the older version, it doesn't work. 
If anyone could help me out working with the pbeis and its implementation.

**Comment by rtimms** [comment, marked as answer]
It’s in the process of being updated [here](https://github.com/pybamm-team/pybamm-eis)
